node/clients/biggy-search.ts

```diff
@@ -269,6 +270,7 @@ export class BiggySearchClient extends ExternalClient {
       fuzzy,
       locale: this.locale,
       bgy_leap: leap ? true : undefined,
+      allowRedirect:  allowRedirect == null ? true : allowRedirect,
```

Comment:
is there any way this object is `null`? in the typings it's boolean or undefined

Reply:
I don't think so, I added this check because also checks if is undefined